src/report.js

```diff
@@ -0,0 +1,24 @@
+import { utcFormat, utcParse } from 'd3-time-format';
+
+const yearFormat = utcFormat('%Y');
+const yearMonthDayFormat = utcFormat('%Y-%m-%d');
+
+/**
+ * @typedef PathProps
+ * @prop {string} reportName
+ * @prop {Date} date
+ * @prop {string=} env 
+ */
+
+/**
+ * @param {PathProps} props
+ */
+function path({ reportName, date, env = 'prod' }) {
```

Comment:
Unclear if this is meant to be used as a component. If yes, should be capitalized "Path". If no, "props" is a bit misleading of an argument name.

Reply:
no this is just a utility function, I just though this was the syntax for annotating a a function's keyword arguments

Reply:
> no this is just a utility function, I just though this was the syntax for annotating a a function's keyword arguments

Just a naming quibble. I might call the object "options" or something? At least if we plan to use React, "props" would make me think of a component.

Reply:
got it! updated in 5e8674b63b8f29ee2197ac1232b130a59200f51a